src/pages/builder/Builder.jsx

```diff
@@ -1,3 +1,170 @@
+        //   'padding': '10px 20px',
+        //   'border': 'none',
+        //   'border-radius': '5px',
+        // });
+
+        editor.BlockManager.add('red-button-block', {
+          label: 'Red Button',
+          category: 'Styled Components',
+          content: `<button class="custom-red-button" style="background-color: red; color: white; padding: 10px 20px; border: none; border-radius: 5px; cursor: pointer;" contentEditable="true">Click Me</button>`,
+        });
+
+        editor.BlockManager.add('input-label-block', {
+          label: 'input-label',
+          category: 'Styled Components',
+          content: `<div style="display: flex; flex-direction: column; align-items: center; font-family: Arial, sans-serif; border: 1px solid #ccc; border-radius: 5px; background-color: #f5f5f5; padding: 20px;">
+          <div style="display: flex; flex-direction: row; align-items: center; margin-bottom: 10px;">
+          <label for="name" style="margin-right: 10px;">Name:</label>
+          <input type="text" id="name" style="width: 100%; padding: 8px; border: 1px solid #ccc; border-radius: 3px; background-color: transparent;"></div>
+        </div>`,
+        });
+        editor.BlockManager.add('centered-form-block', {
+          label: 'Centered Form',
+          category: 'Styled Components',
+          content: `
+          <div style="display: flex; flex-direction: column; align-items: center; font-family: Arial, sans-serif; border: 1px solid #ccc; border-radius: 5px; background-color: #f5f5f5; padding: 20px;">
+          <h1 style="margin-bottom: 20px;">Form</h1>
+          <div style="display: flex; flex-direction: row; align-items: center; margin-bottom: 10px;">
+            <label for="name" style="margin-right: 10px;">Name:</label>
+            <input type="text" id="name" style="width: 100%; padding: 8px; border: 1px solid #ccc; border-radius: 3px; background-color: transparent;">
+          </div>
+          <div style="display: flex; flex-direction: row; align-items: center; margin-bottom: 10px;">
+            <label for="mail" style="margin-right: 10px;">Mail:</label>
+            <input type="text" id="mail" style="width: 100%; padding: 8px; border: 1px solid #ccc; border-radius: 3px; background-color: transparent;">
+          </div>
+          <div style="display: flex; flex-direction: row; align-items: center; margin-bottom: 10px;">
+            <label for="complaint" style="margin-right: 10px;">Complaint:</label>
+            <input type="text" id="complaint" style="width: 100%; padding: 8px; border: 1px solid #ccc; border-radius: 3px; background-color: transparent;">
+          </div>
+          <button style="background-color: green; color: white; padding: 10px 20px; border: none; border-radius: 5px; cursor: pointer;">Submit</button>
+        </div>
+          `,
+        });
+        editor.BlockManager.add('styled-faq', {
+          label: 'Styled FAQ',
+          category: 'Styled Components',
+          content: `
```

Comment:
Let's please add classes to the HTML elements we have here and move the styles to the canvas.css file we created earlier. 🙏🏼